security/container_security/security-container-signature.adoc

```diff
@@ -0,0 +1,17 @@
+[id="security-container-signature"]
+= Container image signatures
+
+Red Hat delivers signatures for the images in the Red Hat Container Registries. Those signatures can be automatically verified when being pulled to {product-title} 4 clusters using the Machine Config Operator (MCO).
```

Comment:
_signatures can be verified_ or _are verified_? If the verification is optional or manual, is there a link you can add here about doing the verification? 
s/using/by using